dev-docs/bidders/smoot.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Smoot
+description: Prebid Smoot Bidder Adapter
+biddercode: smoot
+gpp_sids: usstate_all
+tcfeu_supported: true
```

Comment:
If you claim `tcfeu_supported` you must provide a `gvl_id`

Reply:
My bad! Already corrected